README.md

```diff
@@ -113,90 +113,117 @@ Now we can apply this configuration to the cluster using:
-
-The `cert-manager.io/issuer` annotation tells cert-manager to install a TLS certificate received from the Issuer we installed earlier.
+Now we have everything set up we can issue a Certificate from our Venafi TPP instance. 
+Open the `certificate.yaml` file.
 ```yaml
-apiVersion: extensions/v1beta1
-kind: Ingress
+---
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
 metadata:
-  name: hello-world-ingress
+  name: demo-certificate
   namespace: default
-  annotations:
-    kubernetes.io/ingress.class: "nginx"
-    cert-manager.io/issuer: "venafi-tpp-issuer"
 spec:
-  tls:
-    - hosts:
-        - <place external hostname here>
-      secretName: venafi-demo-tls
-  rules:
-    - host: <place external hostname here>
-    [...]
+  secretName: demo-tls
+  duration: 2160h # 90d
+  dnsNames:
+    - demo.example.com
+  issuerRef:
+    name: venafi-tpp-issuer
+    kind: Issuer
 ```
 
-After changing and saving the file we can deploy it using `kubectl apply -f hello-world.yaml`
-
-Now we have an Ingress deployed with a `cert-manager.io/issuer` annotation, this will tell cert-manager to use a specific issuer to automatically fetch and assign a certificate according to the policy you have configured on your Issuer resource.
-We can see them being issued using:
+Here we see a certificate resource for `demo.example.com` issued by the `venafi-tpp-issuer` we created before.
+We can nog add it to out cluster using:
 ```console
-$ kubectl get certificate
-NAME              READY   SECRET            AGE
-venafi-demo-tls   True    venafi-demo-tls   1m
+$ kubectl apply -f certificate.yaml
 ```
-The private key and certificate have been saved into `venafi-demo-tls` inside the Kubernetes secret store from where NGINX picks it up.
-The NGINX Ingress controller has native support for reading TLS secrets from Kubernetes.
-
-### Testing the deployment
-
-#### Using curl
-You can see it being served using:
+We can see it using:
 ```console
-$ curl https://<hostname> -k -v
-[...]
-* SSL connection using TLSv1.2 / ECDHE-RSA-AES256-GCM-SHA384
-* ALPN, server accepted to use http/1.1
-* Server certificate:
-*  subject: O=cert-manager
-*  start date: Feb 20 14:51:12 2020 GMT
-*  expire date: Feb 19 14:51:12 2021 GMT
-*  issuer: DC=local; DC=traininglab; CN=traininglab-Root-CA
-*  SSL certificate verify result: self signed certificate in certificate chain (19), continuing anyway.
+$ kubectl get certificates
+NAME               READY   SECRET     AGE
+demo-certificate   True    demo-tls   10s
 ```
-Notice that the issue in this example is being signed by a training Venafi TPP instance which is not a trusted certificate authority.
+When creating a `Certificate` resource cert-manager will generate a private key and a Certificate Signing Request. The private key is stored inside the secret we defined in teh configuration: `demo-tls`. This is stored in the Kubernetes secret store.
```

Comment:
nit: comma after `resource`

Reply:
Also `teh` -> `the`

Also maybe write `secret` as `Secret resource` to make it clear?